docs/getting-started/advanced-setting.asciidoc

```diff
@@ -67,6 +67,18 @@ IMPORTANT: {es-sec} requires {ecs-ref}[ECS-compliant data]. If you use third-par
 collectors to ship data to {es}, the data must be mapped to ECS.
 <<siem-field-reference>> lists ECS fields used in {es-sec}.
 
+[discrete]
+[[update-threat-intel-indices]]
+== Update default Elastic Security threat intelligence indices
+
+The `securitySolution:defaultThreatIndex` advanced setting defines threat intelligence indices that {elastic-sec} will use when collecting threat match data. One or more threat intelligence indices can be defined; the `filebeat-*` index is specified by default.
+
+The `securitySolution:defaultThreatIndex` setting controls features that query threat indices, such as the Threat Intelligence view on the Overview page and threat indicator match rules. You can enable Cross Cluster Search (CSS) by adding threat indices from remote clusters.
```

Comment:
This is very similar to the above paragraph; I think these need to be consolidated into one.